code-challenges.test.js

```diff
@@ -64,4 +150,42 @@ const accountTransactions2 = [250, -89, 100, -96]
 const accountTransactions3 = []
 // Expected output: []
 
+
+describe("accuSum", () => {
+  it("Create a function that takes in an array and returns an array of the accumulating sum. An empty array should return an empty array.", () => {
+    const accountTransactions1 = [100, -17, -23, -9];
+    expect(accuSum(accountTransactions1)).toEqual([100, 83, 60, 51]);
+    const accountTransactions2 = [250, -89, 100, -96];
+    expect(accuSum(accountTransactions2)).toEqual([250, 161, 261, 165]);
+    const accountTransactions3 = [];
+    expect(accuSum(accountTransactions3)).toEqual([]);
+  });
+});
+
+// expect(received).toEqual(expected) // deep equality
+
+// Expected: [100, 83, 60, 51]
+// Received: undefined
+
+
 // b) Create the function that makes the test pass.
+
+//PSEUDOCODE:
+//First I will declare a function sumArr and the create a conditional for the empty array, and return empty array.
+// Next, I'll establish the conditional for the numbers in the array, and create a temp array.
+// Next I will add the first number using arr[0] to the second and push to the temp array.
+// I'll utilize the .map method and continue iterating through and return the temp array.
```

Comment:
Your logic and tests look great!  If you want to push yourself to the next level, try to rework these problems to include a higher-order function instead of for loops.  Great work though!